frontend/public/components/search.tsx

```diff
@@ -234,26 +290,37 @@ const SearchPage_: React.FC<SearchProps> = (props) => {
       </PageHeading>
       <div className="co-search">
         <Accordion className="co-search__accordion" asDefinitionList={false} noBoxShadow>
-          {[...selectedItems].map((item) => {
-            const isCollapsed = collapsedKinds.has(item);
+          {[...selectedItems].map((resource) => {
+            const isCollapsed = collapsedKinds.has(resource);
             return (
-              <AccordionItem key={item}>
+              <AccordionItem key={resource}>
                 <AccordionToggle
-                  onClick={() => toggleKindExpanded(item)}
+                  className="co-search-group__accordion-toggle"
+                  onClick={() => toggleKindExpanded(resource)}
                   isExpanded={!isCollapsed}
-                  id={`${item}-toggle`}
+                  id={`${resource}-toggle`}
                 >
-                  {getToggleText(item)}
+                  {getToggleText(resource)}
+                  <a
```

Comment:
This should be a button.

Reply:
yeah, but then we have a button as a child of a button which gives console warning:
```
Warning: validateDOMNesting(...): <button> cannot appear as a descendant of <button>
```